src/box_robot/control/control.yaml

```diff
@@ -0,0 +1,38 @@
+MYROBOT:
+  # Publish all joint states -----------------------------------
+  joint_state_controller:
+    type: joint_state_controller/JointStateController
+    publish_rate: 250  
+
+  # Position Controllers ---------------------------------------
+  joint1_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: joint2
+    pid: {p: 1000.0, i: 0.1, d: 1.0}
+
+  joint2_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: joint3
+    pid: {p: 1000.0, i: 0.1, d: 1.0}
+
+  joint3_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: joint4
+    pid: {p: 1000, i: 0.1, d: 1.0}
+  joint4_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: joint5
+    pid: {p: 1000, i: 0.1, d: 1.0}
+  
+  joint5_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: joint6
+    pid: {p: 1000, i: 0.1, d: 1.0}
+  joint6_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: leg_to_knee_l
```

Comment:
very strange naming principle. Can you make it more fast-forward, understandable?

Reply:
i have changed the name of joints ,